keps/140-scenario-based-simulation/README.md

```diff
@@ -298,10 +320,13 @@ ScenarioStep is:
 The following shows what happens at a single step in ScenarioStep:
 
 1. run all operations defined for that step.
-2. scheduler starts scheduling.
-3. scheduler stops scheduling when it can no longer schedule any more Pods.
-4. update status.scenarioResult.
-5. move to next step.
+2. the scenario controller changes status.SchedulerStatus to SchedulerWillRun.
+3. the scheduler starts scheduling and changes status.SchedulerStatus to SchedulerRunning.
+4. the scenario controller changes status.SchedulerStatus to SchedulerWillStop, 
+when it can no longer schedule any more Pods.
```

Comment:
In general, I think it would be impossible for the simulator to detect the scheduler finishes the scheduling in this step.  So, we add a note for the assumptions for schedulers that the simulator support.  I think another PR is ok.

Reply:
> In general, I think it would be impossible for the simulator to detect the scheduler finishes the scheduling in this step. 

You mean we cannot support all schedulers in the world because the possibilities for how to implement a scheduler are endless, right?
I added the answer at bottom of the KEP in this PR. ("can scenario work with all kind of schedulers?" section)
Please check it.